src/components/VmsList/index.js

```diff
@@ -51,7 +51,7 @@ const VmsList = ({ vms, requestActive }) => {
     el = <NoVmAvailable />
   }
 
-  return <div className='route-page-component'>{el}</div>
+  return el
```

Comment:
How does removing a wrapper div with a class fix the problem?

Reply:
Because infinite scroller use parent node for tracking scrolling. Thus with this div wrapper it doesn't work, because div has `overflow: visible` by default.

Reply:
So the `div` that contains the `InfiniteScroll` component needs `overflow: auto`.  That makes sense.  I think we can do better so that a change to `#page-router-render-component` won't cause problems in the future.

Reply:
Actually I can't imagine what problems can it cause. Because `#page-router-render-component` has only one goal it must be be scrollable.

Reply:
This change is ok.

Reply:
> Actually I can't imagine what problems can it cause.

Which is why we take simple precautions now...especially since a simple change to add a wrapper div already caused so much havoc.

Reply:
Thanks! I spun this discussion into #716